### PR TITLE
Exactly one round

### LINKED ISSUE #56: Broadcast phase announcement exactly once (stop broadcasting new rounds on errors and other cases)
Based on discussion in #43

It looks like the new round broadcast is not strictly necessary at the beginning of a round so I believe `broadcastNewRound()` method and calls can be removed with no other changes.

@zquestz is there anything else necessary on this one?

### PR DESCRIPTION
**Pull request by emergent-reasons**
Fixes #56 

Is this enough?

Based on #54 `isolate-pools` so it needs to be rebased if that is not merged.
The only relevant commit is 8f57cc0 and anything after.

### CODE REVIEW

File: server/broadcast.go [hunk 1]
```diff
@@ -32,17 +32,13 @@ func (pi *packetInfo) broadcastMessage() error {
 		if vk == broadcastKey {
 			err := pi.broadcastAll(msgs)
 			if err != nil {
-				pi.broadcastNewRound(true)
-
-				// Don't disconnect, we broadcasted a new round.
+				// Don't disconnect
```
Comment (zquestz) [inline server/broadcast.go]: I think we can remove the error return value. We don't return any errors from this method and we can probably change the function signature.

Comment (zquestz) [reply server/broadcast.go]: Probably can change in a follow up PR. =)

File: server/tracker.go [hunk 2]
```diff
@@ -5,8 +5,6 @@ import (
 	"sync"
 	"time"
 
-	"github.com/cashshuffle/cashshuffle/message"
```
Comment (zquestz) [inline server/tracker.go]: EPIC!

File: server/tracker.go [hunk 3]
```diff
@@ -39,14 +34,8 @@ type Tracker struct {
 	verificationKeys        map[string]net.Conn
 	mutex                   sync.RWMutex
 	denyIPMatch             map[ipPair]time.Time
-	pools                   map[int]map[uint32]*playerData
-	poolAmounts             map[int]uint64
-	poolVoters              map[int]int
-	poolVersions            map[int]uint64
-	poolTypes               map[int]message.ShuffleType
+	pools                   map[int]*Pool
```
Comment (zquestz) [inline server/tracker.go]: So fresh and so clean!

**Comment by zquestz** [comment]
This looks great. Merging!

File: server/broadcast.go (hunk 1 above)
Comment (emergent-reasons) [reply server/broadcast.go]: Will do.

**Pull request merged**